bmrg-flow/templates/secrets-cert.yaml

```diff
@@ -0,0 +1,17 @@
+{{- $platform := "bmrg" -}}
+{{- $product := "flow" -}}
+{{- $tier := "certs" -}}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "bmrg.name" (dict "context" $ "tier" $tier ) }}
+  labels:
+    {{- include "bmrg.labels.chart" (dict "context" $ "tier" $tier ) | nindent 4 }}
+type: Opaque
+data:
+  {{- if (eq $values.workers.logging.type "elastic" ) }}
+  ca.crt: {{ $.Values.workers.logging.secret }}
+  {{- end }}
+  {{- if $.Values.global.database.mongodb.tls.enable }}
+  mongodb.crt: {{ $.Values.global.database.mongodb.tls.cert }}
+  {{- end }}
```

Comment:
Hey @gchickma I am not sure I understand this file. I might be mistaken but I thought `$.Values.workers.logging.secret` was a reference to the secret file, not the contents to be put into a Secret file.